specification/http/src/test/java/org/kaazing/specification/http/rfc7235/HeaderFieldDefinitionsIT.java

```diff
@@ -29,49 +38,51 @@
  */
 public class HeaderFieldDefinitionsIT {
 
+    private final K3poRule k3po = new K3poRule().setScriptRoot("org/kaazing/specification/http/rfc7235/headers");
+
+    private final TestRule timeout = new DisableOnDebug(new Timeout(5, SECONDS));
+
+    @Rule
+    public final TestRule chain = outerRule(k3po).around(timeout);
+
     @Test
-    @Ignore("Not Implemented")
-    public void secureServerShouldSend401ToAnyUnAuthorizedRequest() {
-        // this includes 101
-
-        // A server generating a 401 (Unauthorized) response MUST send a
-        // WWW-Authenticate header field containing at least one challenge. A
-        // server MAY generate a WWW-Authenticate header field in other response
-        // messages to indicate that supplying credentials (or different
-        // credentials) might affect the response.
+    @Specification({
+        "invalid.username.valid.password/response",
+        "invalid.username.valid.password/request" })
+    public void shouldRespond401WithInvalidUsernameValidPassword() throws Exception {
+        k3po.finish();
     }
 
     @Test
-    @Ignore("Not Implemented")
-    public void proxyMustNotModifyWWWAuthenticateHeader() {
-
+    @Specification({
+        "invalid.user/response",
+        "invalid.user/request" })
+    public void shouldRespond401ToInvalidUser() throws Exception {
+        k3po.finish();
     }
 
     @Test
-    @Ignore("Not Implemented")
-    public void clientMaySendAuthenticationHeaderToServerAfter401() {
-        // The "Authorization" header field allows a user agent to authenticate
-        // itself with an origin server -- usually, but not necessarily, after
-        // receiving a 401 (Unauthorized) response. Its value consists of
-        // credentials containing the authentication information of the user
-        // agent for the realm of the resource being requested.
+    @Specification({
+        "valid.username.invalid.password/response",
+        "valid.username.invalid.password/request" })
+    public void shouldRespond401WithValidUsernameInvalidPassword() throws Exception {
+        k3po.finish();
     }
```

Comment:
Is there duplication between these tests and the ones in `AccessAuthenticationFrameworkIT`?

If so, then let's eliminate the duplication as we need the minimal set of tests that are necessary and sufficient to get specification coverage.